Source/Core/Updater/Main.cpp

```diff
@@ -626,6 +626,16 @@ bool PerformUpdate(const TodoList& todo, const std::string& install_base_path,
 
 }  // namespace
 
+void FatalError(std::string message)
```

Comment:
This should probably be within the anonymous namespace and also take the string by const reference.